client/src/utils/queryParams-utils.ts

```diff
@@ -9,10 +9,10 @@ export const getQueryParams = (
   let params = { ...initialQueryParams };
   for (const [key, value] of Object.entries(queryParams)) {
     if (!isNil(value)) {
-      if (Array.isArray(value) && value[0] !== '') {
-        params = { ...params, [key]: value[0] };
+      if (Array.isArray(value) && value.join(',') !== '') {
+        params = { ...params, [key]: value.join(',').trim() };
```

Comment:
it would make more sense to apply `trim` on each element, then `join`, then check it against empty string, then save. Otherwise it won't trim spaces in the middle of joined elements 
`['   it is  ', '   a bad value   '].join(',').trim()` -> `it is  ,   a bad value`

Reply:
like this?

`  if (Array.isArray(value)) {
        const trimmedElements = value.map(elem => elem.trim()).join(',');
        if (trimmedElements !== '') {
          params = { ...params, [key]: trimmedElements };
        }`